src/Sentry/Extensibility/HubAdapter.cs

```diff
@@ -66,6 +66,7 @@ public IDisposable PushScope<TState>(TState state)
         /// <summary>
         /// Forwards the call to <see cref="SentrySdk"/>.
         /// </summary>
+        [Obsolete]
```

Comment:
Missing the alternative API:

Something like:

```suggestion
        [Obsolete("Use the overload of the capture method that takes an action to configure a scope")]
```